src/components/QA/Answer.jsx

```diff
@@ -41,19 +45,19 @@ class Answer extends Component {
   render() {
     const {answer} = this.props;
     const {handleReport, handleHelpful} = this;
-    const {answerer_name, date, body, helpfulness, photos} = answer[1];
-    const answerId = answer[0];
+    const {answerer_name, date, body, helpfulness, photos} = answer;
 
-    const images = photos.map((src) => {
+    const images = photos.map((photo) => {
       return (
-        <img src={src} key={src} className='A_Images' />
+        <img src={photo.url} key={photo.id} className='A_Images' />
```

Comment:
Suggestion: I think if you use `object-fit: cover;` on your image classes it should maintain the aspect ratio